src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -833,6 +833,7 @@ static void refda_adm_ietf_dtnma_agent_edd_num_msg_tx_failed(refda_edd_prod_ctx_
      * |START CUSTOM FUNCTION refda_adm_ietf_dtnma_agent_edd_num_msg_tx_failed BODY
      * +-------------------------------------------------------------------------+
      */
+    (void)ctx;
```

Comment:
The callbacks issuing warnings is actually valuable, because these are not yet implemented.